### DISCUSSION TITLE
Lets discuss about why Django is not popular?

### DISCUSSION THREAD

**Discussion by dawit-melka** [community/community#67619] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Why django is not popular

**Comment by dawit-melka** [comment]
I think it is because most web developers are familiar with JavaScript frame works than python

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
